diff --git a/dist/env-api-keys.js b/dist/env-api-keys.js
index 32871067aff425e1fbae02b5740c428140045979..63849cf60b7819e3b28bafde68cdafc9c4ce77a5 100644
--- a/dist/env-api-keys.js
+++ b/dist/env-api-keys.js
@@ -2,7 +2,7 @@
 let _existsSync = null;
 let _homedir = null;
 let _join = null;
-const dynamicImport = (specifier) => import(specifier);
+const dynamicImport = (specifier) => Function("specifier", "return import(specifier)")(specifier);
 const NODE_FS_SPECIFIER = "node:" + "fs";
 const NODE_OS_SPECIFIER = "node:" + "os";
 const NODE_PATH_SPECIFIER = "node:" + "path";
